recipes/smurff/conda_build_config.yaml

```diff
@@ -0,0 +1,9 @@
+hdf5:
```

Comment:
Please remove this file, this (the pinnings) shouldn't be needed when build inside of the conda-forge infrastructure.

Reply:
File removed.